docs/sequenceDiagrams/EditSequenceDiagramCode.txt

```diff
@@ -0,0 +1,26 @@
+title Edit
+
+
+
+EditCommand->+TaskList: editTask()
+
+TaskList->TaskList: getTaskFromNumber()
+activate TaskList
+deactivate TaskList
+
+TaskList->TaskList: createTask()
+activate TaskList
+TaskList->*+Task: <<create>>
```

Comment:
Rest of this looks good, one small suggestion to change this to TaskList->*Task:  , it makes the arrow point directly to the block at the top.

Reply:
This gives me a syntax error for some reason